### PR TITLE
add props static method like Scala one.

### PR DESCRIPTION
**Pull request by zarinfam**
I add props static method like Scala one in Using TCP java documentation.

### CODE REVIEW

File: akka-docs/rst/java/code/docs/io/japi/IODocTest.java [hunk 1]
```diff
@@ -90,6 +90,10 @@ public void onReceive(Object msg) throws Exception {
     
     final InetSocketAddress remote;
     final ActorRef listener;
+    
+    public static Props props(InetSocketAddress remote, ActorRef listener) {
+        return Props.create(TcpClientActor.class, remote, listener);
+    }
```
Comment (ktoso) [inline akka-docs/rst/java/code/docs/io/japi/IODocTest.java]: Good catch, thanks! 
Would you mind doing the same for `Server` and using those `props` methods on lines 157, 159 so they don't show up as unused?

**Comment by ktoso** [comment]
When adding the above small changes could you also please rename the commit message to abide to our commit message style as shown in [contributing.md](https://github.com/akka/akka/blob/master/CONTRIBUTING.md#creating-commits-and-writing-commit-messages) (so in this case it would be `=doc add props static method in IODocTest`)?

Thanks in advance!

**Comment by ktoso** [comment]
OK TO TEST

**Comment by ktoso** [comment]
@zarinfam, failure was:

```
[error] /mnt/akka/jenkinsakka/localhome-a4/workspace/pr-validator-per-commit-jenkins/akka-docs/rst/java/code/docs/io/japi/IODocTest.java:95: cannot find symbol
[error] symbol  : class TcpClientActor
[error] location: class docs.io.japi.IODocTest.Client
[error]         return Props.create(TcpClientActor.class, remote, listener);
[error]         
```

Probably a missing import? Would you mind fixing up this PR (`git commit --amend`) so we can go ahead with merging?

**Comment by rkuhn** [comment]
idle notice: in the absence of further activity we’ll take over next Wednesday

**Comment by rkuhn** [comment]
Taken over due to inactivity.

**Pull request closed without merging**